src/components/StructuredContent/StructuredContent.stories.mdx

```diff
@@ -111,16 +109,36 @@ export const imageSizeEnumValues = getStorybookEnumValues(
         "Nullam non nisi est sit. Turpis egestas integer eget aliquet nibh " +
         "praesent. Tortor at risus viverra adipiscing at. Eu augue ut lectus " +
         "arcu bibendum at varius vel pharetra.",
+      calloutText: "This is the callout text",
+      className: undefined,
+      headingText: "Heading text",
+      id: "structured-content",
+      "imageProps.alt": "Image alt text",
```

Comment:
Why are the `imageProps` props in quotes?

Reply:
It's in quotes so it appears in Storybook's args table and so that the Controls can pick up the changes. Likewise, when getting this value from the `args` object, you have to do `args["imageProps.alt"]` -- which is pretty dumb. I expected `args.imageProps.alt` but this doesn't work in Storybook and others also submitted issues regarding nested values. The default way storybook does this isn't very intuitive so by making it `"imageProps.alt"`, it's hopefully clear that you are updating the `alt` property of the `imageProps` prop object.

Reply:
Got it.  Thanks.  And yes, I thought what was showing in the ArgsTable was clear.